File: ontrack-ui/src/main/java/net/nemerosa/ontrack/boot/properties/CoreExtensionFeature.java
package net.nemerosa.ontrack.boot.properties;

import net.nemerosa.ontrack.extension.support.AbstractExtensionFeature;
import org.springframework.stereotype.Component;

@Component
public class CoreExtensionFeature extends AbstractExtensionFeature {

    public CoreExtensionFeature() {
        super("core", "Core", "Core extensions");
    }

}

File: ontrack-ui/src/main/java/net/nemerosa/ontrack/boot/CoreExtensionFeature.kt
package net.nemerosa.ontrack.boot

import net.nemerosa.ontrack.extension.support.AbstractExtensionFeature

class CoreExtensionFeature : AbstractExtensionFeature(
        "core",
        "Core",
        "Ontrack Core"
) {
    companion object {
        val INSTANCE = CoreExtensionFeature()
    }
}
